src/compiler.rs

```diff
@@ -152,13 +151,9 @@ fn write_polys_file(
 ) {
     for i in 0..degree as usize {
         for (_name, constant) in polys {
-            let mut v = constant[i].clone();
-            if v.sign() == Sign::Minus {
-                // This hardcodes the goldilocks field
-                v += 0xffffffff00000001u64;
-            }
-            file.write_all(&abstract_to_degree(&v).to_le_bytes())
-                .unwrap();
+            let mut bytes = constant[i].to_bytes_be();
```

Comment:
Does this always have the correct length?

Reply:
Why not use `le`?

Reply:
Yes I think so, under the hood it calls `u64::to_le_bytes` which returns `[u8; mem::size_of::<Self>()] = [u8; 8]`